fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -126,21 +136,25 @@ public static Builder newBuilder() {
      * A builder for the scrubbing policy.
      */
     public static class ScrubbingPolicy {
-        public static final ScrubbingPolicy DEFAULT = new ScrubbingPolicy(1000, true, 0, false, false);
+        public static final ScrubbingPolicy DEFAULT = new ScrubbingPolicy(1000, false, 0, false, false, 0, false);
         private final int logWarningsLimit;
         private final boolean allowRepair;
         private final long entriesScanLimit;
         private final boolean ignoreIndexTypeCheck;
         private final boolean useLegacy;
+        private final int rangeId;
+        private final boolean rangeReset;
 
         public ScrubbingPolicy(int logWarningsLimit, boolean allowRepair, long entriesScanLimit,
```

Comment:
To avoid making a breaking change, it would be good ot have an override that does not take the new parameters, and provides default values.

Reply:
Mmm.. this constructor was never meant to be public. It also doesn't have any javadoc, test, or an API example. Would it make sense to convert it to private now?